### ISSUE TITLE
did you see this repo

### ISSUE THREAD

**Issue by mkonwerski** [0x416c6578/0x416c6578.github.io#1]
https://github.com/Aluminum-z/Laser-Radar-LDS-006-Drive-Test

**Comment by 0x416c6578** [comment]
yeah, it was found a few months ago. see https://github.com/0x416c6578/lds-006-firmware/issues/2 . I plan to update / implement / experiment with it when I have some time
